### ISSUE TITLE
Avast YLS

### ISSUE THREAD

**Issue by humpalum** [infosec-intern/vscode-yara#66]
Avast released a Language Server for Yara: https://github.com/avast/yls
Do you think we should implement it? It seems to be actively developed which this extension could profit from. 

Implementation doesn't seem to hard. Avast shows a example in their extension: https://github.com/avast/yls/tree/master/editors/vscode

The first step of installing the Server could be done in an command at startup, which also checks if its up to date. The python extension does that in a similar way.

**Comment by infosec-intern** [comment]
Yeah this is a very cool project. I previously tried building something similar but this is far more feature-packed, and the code looks much nicer

The only issue that immediately comes to mind is using it on Windows systems where Python is not pre-installed, but I don't know if that's actually a practical issue
